platform/pulpcore/app/models/repository.py

```diff
@@ -125,9 +103,16 @@ class Importer(ContentAdaptor):
     ON_DEMAND = 'on_demand'
     BACKGROUND = 'background'
     DOWNLOAD_POLICIES = (
-        (IMMEDIATE, 'Download Immediately'),
-        (ON_DEMAND, 'Download On Demand'),
-        (BACKGROUND, 'Download In Background'))
+        (IMMEDIATE, 'Sync task is not complete until downloading is finished.'),
+        (ON_DEMAND, 'Sync task updates metadata but does not download content until requested.'),
+        (BACKGROUND, 'Sync task updates metadata and completes. Downloads complete in background.'))
+
+    # Sync Modes
+    ADDITIVE = 'additive'
+    MIRROR = 'mirror'
+    SYNC_MODES = (
+        (ADDITIVE, 'Add new content, keep content that was removed upstream.'),
+        (MIRROR, 'Add new content, remove content was removed upstream.'))
```

Comment:
Suggest something like:

ADDITIVE:  _Add new content._

MIRROR: _Add new content and remove content that is no longer in the remote repository._

For additive, some content could still be removed from the repository in cases such as RPM only keeping X versions of RPMs.